test/constrained-generics/hashable/user-hash-no-implements.good

```diff
@@ -0,0 +1,2 @@
+user-hash-no-implements.chpl:15: warning: R has a hash function that is being used by the standard library. However, R does not implement hashable. In the future, this will result in an error.
```

Comment:
Recommend putting the names in single quotes

```
warning: 'R' has a hash function that is being used by the standard library. However, 'R' does not implement 'hashable'. In the future, this will result in an error.
```

Should it say something about how to implement hashable, e.g.

```
user-hash-no-implements.chpl:3: note: change it to `record R : hashable` to implement hashable
```